lib/hashie/extensions/dash/indifferent_access.rb

```diff
@@ -11,6 +11,7 @@ module ClassMethods
           # Check to see if the specified property has already been
           # defined.
           def property?(name)
+            name = defined?(translations) && translation_exists?(name) ? translations[:"#{name}"] : name
```

Comment:
I just think it's safer and more explicit, because I might as well have `translations` doing something else in there.